package.json

```diff
@@ -43,7 +43,7 @@
   "scripts": {
     "bump": "lerna publish --skip-npm --skip-git && lerna-changelog",
     "precommit": "lint-staged",
-    "release": "lerna exec --bail=false -- 'can-npm-publish && npm publish'",
+    "release": "lerna exec --bail=false -- 'can-npm-publish && npm publish && echo done ; echo skip'",
```

Comment:
`can-npm-publish` exit code  will return 1, if not publish target.
In the case of that, although set `--bail=false` option, `lerna exec` will return 1.